testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -568,6 +623,11 @@ def __update_bindings_with_subsystem_configuration(self, agent):
       except KeyError:
         pass
       self.bindings[key] = value
+    
+    if not self.bindings['SPINNAKER_AZURE_ACCOUNT']:
```

Comment:
This should be added automatically in head (in the base provider support class that you inherit the AzureScenarioSupport from)